src/pybind/mgr/dashboard/frontend/src/app/ceph/block/iscsi-target-list/iscsi-target-list.component.html

```diff
@@ -14,6 +14,7 @@
   </ng-container>
 </cd-alert-panel>
 
+<legend i18n>Targets</legend>
```

Comment:
I guess you used `legend` because other tables did it? Supposedly [`<legend>`](https://www.w3schools.com/tags/tag_legend.asp) should be used within `<fieldset>`...

I'd use:
```suggestion
<h1 i18n>Targets</h1>
```
Or:
```suggestion
<header>
  <h1 i18n>Targets</h1>
</header>
```

Reply:
Yes. Sounds good, but just will use h3 or higher for text size (as h1 seems too large!)

Reply:
Well, if `<h1>` is too large... let's restyle it! `<h[1-6]>` tags [should be added by keeping the hierarchy](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements) in the HTML doc (and not used based on "this looks too large/small for this"). Basically, the first heading should be `<h1>`, next `<h2>`, ...

With a document made up from different components this is really hard to manage that... We currently have:

```html
ceph/dashboard/info-card/info-card.component.html:<h4 class="card-title m-4">
ceph/cluster/osd/osd-creation-preview-modal/osd-creation-preview-modal.component.html:<h4 i18n>DriveGroups</h4>
ceph/block/rbd-configuration-form/rbd-configuration-form.component.html:<h4 class="cd-header">
core/forbidden/forbidden.component.html:<h1 i18n>Forbidden</h1>
core/forbidden/forbidden.component.html:<h2 i18n>Sorry, you are not allowed to see what you were looking for.</h2>
core/not-found/not-found.component.html:<h1 i18n>Sorry, we could not find what you were looking for.</h1>
core/auth/sso/sso-not-found/sso-not-found.component.html:<h1 i18n>Sorry, the user does not exist in Ceph.</h1>
core/auth/sso/sso-not-found/sso-not-found.component.html:<h4 i18n>
core/auth/login-password-form/login-password-form.component.html:<h2 i18n>Please set a new password.</h2>
core/auth/login-password-form/login-password-form.component.html:<h4 i18n>You will be redirected to the login page afterwards.</h4>
core/navigation/about/about.component.html:<h3>
shared/components/modal/modal.component.html:<h4 class="modal-title float-left">
shared/components/notifications-sidebar/notifications-sidebar.component.html:<h6 class="card-title bold">{{ executingTask.description }}</h6>
shared/components/notifications-sidebar/notifications-sidebar.component.html:<h6 class="card-title bold">{{ notification.title }}</h6>
```

A possibility would be to mimic [what XHTML does for this](https://www.w3.org/TR/xhtml2/mod-structural.html#edef_structural_h), and create a single `<h>` (e.g.: `<cd-heading>`):
```html
<cd-heading>This is a top level heading</cd-heading>
<p>....</p>
<section>
    <p>....</p>
    <cd-heading>This is a second-level heading</cd-heading>
    <p>....</p>
    <cd-heading>This is another second-level heading</cd-heading>
    <p>....</p>
</section>
```
This can be easily styled:
```css
:root cd-heading {}
:root section cd-heading {}
...
```